src/app/components/logIn/Login.tsx

```
@@ -0,0 +1,154 @@
+	};
+
+	//로그인 함수. 인풋 유효성 검사 미통과 시에 에러 메시지 설정
+	const handleSubmit = async (e: React.FormEvent) => {
+		e.preventDefault();
+		try {
+			//로그인 성공. 에러 메시지 초기화. 이전 페이지로 돌아감감
+			await signinUser({ email: id, password: password });
+			setErrorId('');
+			setErrorPassword('');
+			router.back();
+		} catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)